### ISSUE TITLE
Config path no longer loaded

### ISSUE THREAD

**Issue by jonathandturner** [nushell/nushell#3070]
**Describe the bug**
The path from the `config` is no longer loaded in 0.27

**To Reproduce**
1. Set up some path entries in the config
2. Cry
3.

**Expected behavior**
We should see the path from the config as the top priority paths in the final PATH and in `echo $nu.path`.

**Configuration (please complete the following information):**

Run `version | pivot` and paste the output to show OS, features, etc.

```
17/02/2021 10:41:14> version | pivot
────┬────────────────────┬─────────────────────────────────────────────────────────────────────────────────────────
 #  │      Column0       │                                         Column1
────┼────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────
  0 │ version            │ 0.27.0
  1 │ branch             │ main
  2 │ short_commit       │ 892aae26
  3 │ commit_hash        │ 892aae267d3f6678532cdaf13ce517fcd4111106
  4 │ commit_date        │ 2021-02-17 09:02:13
  5 │ build_os           │ windows-x86_64
  6 │ rust_version       │ rustc 1.50.0 (cb75ad5db 2021-02-10)
  7 │ rust_channel       │ stable-x86_64-pc-windows-msvc (default)
  8 │ cargo_version      │ cargo 1.50.0 (f04e7fab7 2021-02-04)
  9 │ pkg_version        │ 0.27.0
 10 │ build_time         │ 2021-02-17 10:06:44
 11 │ build_rust_channel │ release
 12 │ features           │ clipboard-cli, ctrlc, default, directories, dirs, ichwh, ptree, rustyline, term, trash,
    │                    │ uuid, which, zip
────┴────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────```

**Comment by jonathandturner** [comment]
Caused by:  #3056

**Comment by fdncred** [comment]
`Cry` - lolz

**Comment by Ryan-McLemon** [comment]
Oh wow, just came to report this, you guys are fast!

**Comment by jonathandturner** [comment]
Currently testing this fix: https://github.com/nushell/nushell/pull/3072

If everything is good, we'll release a hotfix release

**Comment by jonathandturner** [comment]
Fixed by #3072 and released as 0.27.1
